packages/checkout/sdk-sample-app/src/components/Sell.tsx

```diff
@@ -22,12 +22,19 @@ export default function Sell({ checkout, provider }: SellProps) {
   const [listingTypeError, setListingTypeError] = useState<string>('');
   const [amount, setAmount] = useState<string>('');
   const [amountError, setAmountError] = useState<string>('');
+  const [expiry, setExpiry] = useState<string | undefined>(undefined);
+  const [expiryError, setExpiryError] = useState<string>('');
   const [tokenAddress, setTokenAddress] = useState<string>('');
   const [contractAddressError, setContractAddressError] = useState<string>('');
   const [error, setError] = useState<any>(null);
   const [success, setSuccess] = useState<boolean>(false);
   const [loading, setLoading] = useState<boolean>(false);
 
+  const isDateValid = (dateStr: string) => {
+    var dateRegex = /^[0-9]{4}-[0-9]{1,2}-[0-9]{1,2}$/;
+    return dateRegex.test(dateStr);
```

Comment:
Can we update the regex so it optionally allows for adding the time part of the date string as well so we can test time works correctly too (I tested it out and all works fine but I think this is something the regex should still accept)? 

Currently the regex is just accepting `YYYY-MM-DD`

I believe the regex should accept a string like `2024-01-19T08:00:00.000Z` and just the date by. itself `2024-01-19`

Reply:
<img width="992" alt="Screenshot 2024-01-18 at 2 51 24 pm" src="https://github.com/immutable/ts-immutable-sdk/assets/4301501/153551a5-0543-4f21-a233-2f5dd4342e94">
<img width="666" alt="Screenshot 2024-01-18 at 2 50 59 pm" src="https://github.com/immutable/ts-immutable-sdk/assets/4301501/12026dbd-c90f-4eb9-be46-e8c65fa24872">